olive/passes/onnx/qnn/qnn_mixed_precision_overrides.py

```diff
@@ -0,0 +1,217 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import json
+from logging import getLogger
+from pathlib import Path
+from typing import Any, Dict, Union
+
+from olive.hardware import AcceleratorSpec
+from olive.model import ONNXModelHandler
+from olive.model.utils import resolve_onnx_path
+from olive.passes.olive_pass import Pass
+from olive.passes.onnx.common import get_external_data_config, model_proto_to_file
+from olive.passes.pass_config import PassConfigParam
+
+logger = getLogger(__name__)
+
+
+class QNNMixedPrecisionOverrides(Pass):
+    """Qnn mixed precision overrides pass.
+
```

Comment:
There is nothing specific about QNN here. These overrides could be used by non-QNN also.

Reply:
Removed qnn.